pkg/volume/gce_pd/gce_util.go

```diff
@@ -149,11 +150,60 @@ func (mounter *gceSafeFormatAndMount) Mount(source string, target string, fstype
 	}
 	args = append(args, options...)
 	args = append(args, source, target)
-	glog.V(5).Infof("exec-ing: /usr/share/google/safe_format_and_mount %v", args)
-	cmd := mounter.runner.Command("/usr/share/google/safe_format_and_mount", args...)
+	safeFormatAndMountPath := "/usr/share/google/safe_format_and_mount"
+	glog.V(5).Infof("exec-ing: %s %v", safeFormatAndMountPath, args)
+	cmd := mounter.runner.Command(safeFormatAndMountPath, args...)
 	dataOut, err := cmd.CombinedOutput()
 	if err != nil {
-		glog.V(5).Infof("error running /usr/share/google/safe_format_and_mount\n%s", string(dataOut))
+		glog.V(5).Infof("error running %s\n%s", safeFormatAndMountPath, string(dataOut))
+		// if safe_format_and_mount is not available on this system use linux utilities directly.
```

Comment:
Why support both ways?  If this is equivalent, then we should drop the external dependency.